README.md

```diff
@@ -23,14 +23,14 @@ the latest release.
 *Assumes you have some understanding of git .*
 1. Clone the repository* to your local machine.
 2. From the base directory, run `pip install -r requirements.txt`. Optionally, create a `venv`.
-3. From the `/doc_src`, run `mkdocs build` to build the documentation.
-4. From `/src` run `python hexsheets.py` to start HexSheets.
+3. From the `/doc_src`, run `mkdocs build` to build the documentation (under the `/src/docs/` folder.)
+4. From `/src` run `python hex-spreadsheet.py` to start HexSheets.
```

Comment:
`hexsheets.py` is actually correct - in the most recent `develop` version I changed the name of that file to match what I've settled on for the name of the project: "HexSheets." I've changed the name of the repo as well (from hex-spreadsheet), but there might actually be a couple of other spots that need to change still.